uPortal-webapp/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -435,7 +435,8 @@
           </xsl:variable>
           <xsl:variable name="permissionChannelId">PORTLET_ID.<xsl:value-of select="@chanID"/></xsl:variable>
           <xsl:variable name="canConfigure">
-            <!-- This option is special in that it evaluates both whether (1) the portlet supports CONFIG mode and (2) this user is allowed to access it. -->
+            <!-- This option is special in that it evaluates:
+             whether the portlet supports CONFIG mode AND if this user is a Portal Admin (or in a principal that has been granted permission) -->
```

Comment:
Is this language somehow clearer?  (I think it's less.)

Reply:
Good point - I'll revert that change.